plugins/modules/backup_plan.py

```diff
@@ -542,13 +614,21 @@ def main():
     client = module.client("backup")
     state = module.params["state"]
     plan_name = module.params["backup_plan_name"]
+
     plan = {
         "backup_plan_name": module.params["backup_plan_name"],
         "rules": [scrub_none_parameters(rule) for rule in module.params["rules"] or []],
         "advanced_backup_settings": [
             scrub_none_parameters(setting) for setting in module.params["advanced_backup_settings"] or []
         ],
     }
+
+    if module.params["rules"]:
+        if module.botocore_at_least("1.31.36"):
+            for each in plan["rules"]:
+                if not each.get("schedule_expression_timezone"):
+                    each["schedule_expression_timezone"] = "Etc/UTC"
```

Comment:
The botocore error won't be particularly friendly for botocore <  1.31.36 if someone passes schedule_expression_timezone